Tasks/ServiceFabricPowerShell/task.json

```diff
@@ -0,0 +1,90 @@
+{
+    "id": "816979F7-C273-4347-9A55-845B721D82CB",
+    "name": "ServiceFabricPowerShell",
+    "friendlyName": "Service Fabric PowerShell",
+    "description": "Run a PowerShell script within the context of an Azure Service Fabric cluster connection.",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=841538)",
+    "category": "Utility",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Agent",
+        "MachineGroup"
```

Comment:
Good catch

---
In reply to: [99755042](https://github.com/Microsoft/vsts-tasks/pull/3527#discussion_r99755042) [](ancestors = 99755042)